src/main/kotlin/lotto/model/LottoNumbers.kt

```diff
@@ -1,9 +1,23 @@
 package lotto.model
 
-class LottoNumbers(private val lottoNumbers: List<LottoNumber>) : List<LottoNumber> by lottoNumbers {
+import java.util.TreeSet
+
+open class LottoNumbers(private val lottoNumbers: Set<LottoNumber>) : Set<LottoNumber> by lottoNumbers {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class LottoNumbers(private val lottoNumbers: TreeSet<LottoNumber>) : Set<LottoNumber> by lottoNumbers {
```